### ISSUE TITLE
Should an exception be thrown if StartAcqusition is not present

### ISSUE THREAD

**Issue by jonnew** [neurogears/onix-refactor#35]
I was testing some things with the library, and I built this workflow. It ran without error. However, because its missing a **StartAcquisition** node, it didn't do anything. Is this a desirable behavior or should this workflow throw an exception indicating that Acquisition is not started?

![image](https://github.com/neurogears/onix-refactor/assets/1429475/951881cd-5c68-46c8-b44c-868bf70a426e)

**Comment by glopesdev** [comment]
@jonnew We could throw on subscription if any nodes waiting on data detect that the context is not running. However, this would prevent more advanced scenarios where the start of acquisition is deferred until a specific moment.

My feeling is that the context not starting is a salient enough effect that it will be easily detected, since no data stream will be streaming, and we should not throw in this case.

**Comment by jonnew** [comment]
Fair enough. I'm fine leaving this alone
